quinn-h3/src/frame.rs

```diff
@@ -5,15 +5,15 @@ use bytes::{Buf, BufMut, Bytes};
 use quinn_proto::coding::{BufExt, BufMutExt, Codec, UnexpectedEnd};
 use quinn_proto::varint;
 
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Clone)]
```

Comment:
Why do we need the `Clone` derive here (and elsewhere)?

Reply:
Maybe it was useful for testing, but smells like a bad copy / paste.

Reply:
Hmm, maybe it's not actually needed. To me `return` always seems like more of a statement, and so it's required to be a in a block. But of course it's actually an expression that returns the never type, so maybe this is just me being stuck in my old ways.